tests/Integration/Integration.Tests/PurchaseOrderModuletests/WhenGettingPurchaseOrder.cs

```diff
@@ -1,13 +1,12 @@
-﻿using Linn.Common.Facade;
-
-namespace Linn.Purchasing.Integration.Tests.PurchaseOrderModuleTests
+﻿namespace Linn.Purchasing.Integration.Tests.PurchaseOrderModuleTests
```

Comment:
the folder name has a lowercase t for test

Reply:
That's odd, I'm not seeing a lower letter locally (but also no changes).

Reply:
You won't have seen it in your Windows filesystem since it's case insensitive... You had merged a whole new PurchaseOrderModule**t**ests (in addition to Iains PurchasOrderModule**T**ests) folder into main since your casing was wrong, but windows was none the wiser. I fixed it with #149 , but one to watch for next time.